protocol/communities_messenger_test.go

```diff
@@ -3585,3 +3654,27 @@ func (s *MessengerCommunitiesSuite) TestHandleImport() {
 	s.Require().NotNil(chat)
 	s.Require().Equal(0, int(chat.UnviewedMessagesCount))
 }
+
+func (s *MessengerCommunitiesSuite) TestJoinedCommunityMembersSharedAddress() {
+	community := s.createCommunity()
+	s.advertiseCommunityTo(community, s.alice)
+	s.advertiseCommunityTo(community, s.bob)
+
+	s.joinCommunity(community, s.alice, AlicePassword)
+	s.joinCommunity(community, s.bob, BobPassword)
+
+	joinedCommunities, err := s.admin.communitiesManager.Joined()
+	s.Require().NoError(err)
+	s.Require().Len(joinedCommunities, 1)
+	community = joinedCommunities[0]
+	s.Require().NotNil(community)
```

Comment:
Another way to do this would be to do `s.admin.GetCommunityByID(community.ID())`

Reply:
Yes, i just what to check here we are actually joined

Reply:
`joined()` will return the communities you're joined. But admin has created that community, so it will always be joined.

Reply:
fixed